scripts/generate-app-config/generateSettingsConfig.js

```diff
@@ -11,9 +11,9 @@ const generateUtilitiesConfig = (appConfig) => {
     defaultTheme: appConfig.defaultTheme,
     themes: appConfig.themes || {},
     defaultLocale: appConfig.defaultLocale,
-    locales: appConfig.locales || {},
-    defaultDirection: appConfig.defaultDir,
-    directions: ['ltr', 'rtl'],
+    locales: appConfig.locales || [],
+    defaultDirection: appConfig.defaultDirection,
+    directions: appConfig.bidirectional ? ['ltr', 'rtl'] : [],
```

Comment:
Lets just default this to ['ltr', 'rtl'] and remove the bidirectional config from appConfig. I don't think there is a good argument for allowing people to turn this off.

Reply:
this actually fixes the next issue I was going to point out 😋 If you turn off config options the settings modal still launches with nothing in it

Reply:
@mjhenkes That's a good point. I have addressed it here: https://github.com/cerner/terra-dev-site/pull/210/commits/1ddcb551a692c13d2a807a172422a79da3579c08